environment/ssh.rst

```diff
@@ -71,89 +78,248 @@ this::
+option when it appears.
 
 .. figure:: ssh-img/install-ssh-win10-5.png
 
-You should see a window that looks like this, with the heading *Optional features*.
+You should see a window that looks like this, with the heading *Optional
+features*.
 
 .. figure:: ssh-img/install-ssh-win10-6.png
 
-Scroll through the list of *Installed features*. If *OpenSSH Client* appears in the list, you are done with this step. Otherwise, click on *+ Add a feature* at the top of the page. You will get a pop-up window with the heading *Add an optional feature*. Start typing *OpenSSH Client*. When the option appears, click on the checkbox next to it.
+Scroll through the list of *Installed features*. If *OpenSSH Client* appears in
+the list, you are done with this step. Otherwise, click on *+ Add a feature* at
+the top of the page. You will get a pop-up window with the heading *Add an
+optional feature*. Start typing *OpenSSH Client*. When the option appears, click
+on the checkbox next to it.
 
 .. figure:: ssh-img/install-ssh-win10-7.png
 
-Then click on the button labeled *Install (1)*. Wait for the progress bar to fill.
+Then click on the button labeled *Install (1)*. Wait for the progress bar to
+fill.
 
 .. figure:: ssh-img/install-ssh-win10-8.png
 
-The installation is complete. You should now re-try the instructions at the top of this page.
+The installation is complete. You should now re-try the instructions at the top
+of this page.
 
 
 Linux
 ~~~~~
 
-Linux systems typically include an SSH client but may occasionally not include one.
-The exact process to install the client may vary from one Linux distribution to another,
-but the following commands should allow you to install SSH in some of the most popular
-Linux distributions:
+Linux systems typically include an SSH client but may occasionally not include
+one. The exact process to install the client may vary from one Linux
+distribution to another, but the following commands should allow you to install
+SSH in some of the most popular Linux distributions:
 
 - **Debian/Ubuntu**: Run ``sudo apt-get install openssh-client``
 - **RHEL/Fedora/CentOS**: Run ``sudo yum install openssh-clients``
 
-Once you've installed SSH, you should re-try the instructions at the top of the page.
+Once you've installed SSH, you should re-try the instructions at the top of the
+page.
+
+
+.. _passwordless-ssh:
+
+Setting up Passwordless SSH
+===========================
+
+You may have noticed that every time you connect to the Linux server with SSH,
+either in a terminal window or with VSCode, you are prompted for your password.
+
+These instructions will allow you to configure your computer to connect to the
+Linux server without being prompted for your password each time, which is much
+more convenient. 
+
+You only need to follow these instructions once on your personal
+computer/laptop.
+
+.. warning :: 
+  If you have already completed the GitHub SSH setup instructions, you
```

Comment:
Add "(from the GitHub Basics tutorial)" after "setup instructions" (with a link to the GitHub Basics tutorial)